packages/braintree-utils/src/braintree-script-loader.ts

```diff
@@ -37,7 +35,7 @@ export default class BraintreeScriptLoader {
             `//js.braintreegateway.com/web/${this.braintreeSdkVersion}/js/client.min.js`,
         );
 
-        if (!this.braintreeHostWindow.braintree?.client) {
+        if (!this.braintreeHostWindow.braintree || !this.braintreeHostWindow.braintree.client) {
```

Comment:
Why do you change this condition? As far as I understand it is the same as prev one